tests/unit/v1/test_cross_language.py

```diff
@@ -125,11 +106,17 @@ def _run_testcase(testcase, call, firestore_api, client):
             call()
     else:
         call()
+
+        wrapped_writes = [write.Write(write_pb) for write_pb in testcase.request.writes]
```

Comment:
Hmm, maybe this should use `Message.wrap`?  E.g.:
```suggestion
        wrapped_writes = [write.Write.wrap(write_pb) for write_pb in testcase.request.writes]
```

Reply:
is this the cause of bad __eq__?

Reply:
Yes.  The conftest protos have the expected requests laid out as "raw" proto-generated classes, which don't compare equal to our wrapped classes (asymmetric).